CRM/Contribute/BAO/Query.php

```diff
@@ -298,7 +150,7 @@ public static function whereClauseSingle(&$values, &$query) {
     list($name, $op, $value, $grouping, $wildcard) = $values;
 
     $quoteValue = NULL;
-    $fields = array_merge(CRM_Contribute_BAO_Contribution::fields(), self::getFields());
+    $fields = self::getFields();
```

Comment:
this was just to add in the missing payment_instrument_id - I added that to the exportables & so not needed